common/src/main/java/tachyon/exception/ExceptionMessage.java

```diff
@@ -100,7 +100,17 @@
   HDFS_FILE_NOT_FOUND("File {0} with id {1} is not found"),
 
   // file system master
+  FILE_MUST_HAVE_VALID_PARENT("{0} does not have a valid parent"),
```

Comment:
I don't think this is a real error (doesn't seem logically possible), it happens if the `InodeTree` is corrupted and assigns an `InodeFile` a child.